### Description
A list of useful tools for genome-wide association analysis. Tools mostly developed &amp; maintained by friends and colleagues. Tools that I have frequently used.

### Links
1. [EPACTS](https://genome.sph.umich.edu/wiki/EPACTS) - versatile software pipeline to perform various statistical tests for identifying genome-wide association from sequence data through a user-friendly interface. 
It includes popular statistical genetics methods like EMMAX, SKAT etc.

2. [gwasTools](https://github.com/ilarsf/gwasTools) - a collection of R scripts for plotting GWAS results (Power plot, Manhattan plot, QQ plot)

3. [LocusZoom](http://locuszoom.sph.umich.edu/) - a suite of interactive tools to provide fast visualization of GWAS results for research and publication.

4. [METAL](https://genome.sph.umich.edu/wiki/METAL_Documentation) - software for meta-analysis of large datasets (such as several whole genome scans) in a convenient, rapid and memory efficient manner.

5. [RMMLR](http://www.biostat.umn.edu/~saonli/publication.htm) - software for rapid genome-wide gene-based association tests with multivariate traits.

6. [SWISS](https://github.com/welchr/swiss) - software to help identify overlap between association scan results and GWAS hit catalogs.
It can also be used to obtain a set of independent (LD-pruned) loci from a given set of loci.

